templates/repo/issue/view_content/pull.tmpl

```diff
@@ -86,22 +79,17 @@
 					<div class="ui divider"></div>
 					<div class="review-item">
 						<div class="review-item-left">
+							<span class="gt-mr-3 text {{.Review.HTMLTypeColorName}}">
+								{{svg (printf "octicon-%s" .Type.Icon) 18}}
```

Comment:
I see `{{svg (printf "octicon-%s" .Review.Type.Icon) 16}}` and `{{svg "octicon-x" 20}}` above. Here is 18?

Reply:
Hmm might need to review again these sizes. Will try to revert them all to 16px I guess. The issue is just that the `sync` octicon appears rather large compared to the `x` one at 16px. Maybe it's sufficient to just increase size of `x`.

Reply:
Reverted all to 16px, except dismiss "x" which is 20 px  because it looks awefully small otherwise. Before:

<img width="88" alt="Screenshot 2023-05-10 at 00 14 32" src="https://github.com/go-gitea/gitea/assets/115237/93a994de-321f-446e-a5d1-ec7dbd7eaa1e">

After:

<img width="94" alt="Screenshot 2023-05-10 at 00 15 02" src="https://github.com/go-gitea/gitea/assets/115237/f8ff1e2b-d913-4fd8-bab1-7bec0d066541">